tests/libs/test_pygit2_utils.py

```diff
@@ -0,0 +1,41 @@
+from unittest import TestCase
+from shaker.libs import pygit2_utils
+from mock import patch
+import pygit2
+
+
+class TestPygit2Utils(TestCase):
+
+    def test_pygit2_check_ssh(self):
+        """
+        TestPygit2Utils:test_pygit2_check_ssh: Check for successful ssh support
+        """
+        pygit2.features = 7
```

Comment:
Magic numbers are bad. This should be expressed in terms of `pygit2.features = pygit2.GIT_FEATURE_SSH & pygit2.GIT_FEATURE_SOMETHINGELSE`.

Also this will change the feature flags for the entire rest of the test run - so we should consider using mock/patch it so that the scope of the change is limited to just this test case.